identity/signer.go

```diff
@@ -0,0 +1,44 @@
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+)
+
+type Signer interface {
+	Sign(message []byte) ([]byte, error)
+}
+
+type keystoreSigner struct {
+	keystoreManager keystoreInterface
```

Comment:
Just noticed. `Signer` reminds very much structure `keystoreWallet` from `go-ethereum/accounts/accounts`

Reply:
Yep, but keystoreWallet is private.